From 2ffc9b1fe3cbba317196199b096d41186e4e675e Mon Sep 17 00:00:00 2001
From: HazouPH <jgrharbers@gmail.com>
Date: Thu, 23 Oct 2014 21:18:31 +0200
Subject: [PATCH] Temp remove of build tinyalsa from source.

Tinyalsa is missing some code at the moment to work with the smi audio
blobs.

Change-Id: I9adb39ccd5eb9af9e996d2c197d668d530122ccb
---
 Android.mk | 56 --------------------------------------------------------
 1 file changed, 56 deletions(-)
 delete mode 100644 Android.mk

diff --git a/Android.mk b/Android.mk
deleted file mode 100644
index 4f6db22..0000000
--- a/Android.mk
+++ /dev/null
@@ -1,56 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES:= external/tinyalsa/include
-LOCAL_SRC_FILES:= mixer.c pcm.c
-LOCAL_MODULE := libtinyalsa
-LOCAL_SHARED_LIBRARIES:= libcutils libutils
-LOCAL_MODULE_TAGS := optional
-LOCAL_PRELINK_MODULE := false
-
-include $(BUILD_SHARED_LIBRARY)
-
-ifeq ($(HOST_OS), linux)
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES:= external/tinyalsa/include
-LOCAL_SRC_FILES:= mixer.c pcm.c
-LOCAL_MODULE := libtinyalsa
-LOCAL_STATIC_LIBRARIES:= libcutils libutils
-include $(BUILD_HOST_STATIC_LIBRARY)
-endif
-
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES:= external/tinyalsa/include
-LOCAL_SRC_FILES:= tinyplay.c
-LOCAL_MODULE := tinyplay
-LOCAL_SHARED_LIBRARIES:= libcutils libutils libtinyalsa
-LOCAL_MODULE_TAGS := optional
-
-include $(BUILD_EXECUTABLE)
-
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES:= external/tinyalsa/include
-LOCAL_SRC_FILES:= tinycap.c
-LOCAL_MODULE := tinycap
-LOCAL_SHARED_LIBRARIES:= libcutils libutils libtinyalsa
-LOCAL_MODULE_TAGS := optional
-
-include $(BUILD_EXECUTABLE)
-
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES:= external/tinyalsa/include
-LOCAL_SRC_FILES:= tinymix.c
-LOCAL_MODULE := tinymix
-LOCAL_SHARED_LIBRARIES:= libcutils libutils libtinyalsa
-LOCAL_MODULE_TAGS := optional
-
-include $(BUILD_EXECUTABLE)
-
-include $(CLEAR_VARS)
-LOCAL_C_INCLUDES:= external/tinyalsa/include
-LOCAL_SRC_FILES:= tinypcminfo.c
-LOCAL_MODULE := tinypcminfo
-LOCAL_SHARED_LIBRARIES:= libcutils libutils libtinyalsa
-LOCAL_MODULE_TAGS := optional
-
-include $(BUILD_EXECUTABLE)
-- 
1.9.1

